crates/router/src/connector/stripe/transformers.rs

```diff
@@ -218,6 +219,8 @@ pub struct ChargesRequest {
     pub currency: String,
     pub customer: Secret<String>,
     pub source: Secret<String>,
+    #[serde(flatten)]
+    pub meta_data: Option<HashMap<String, String>>,
```

Comment:
We are using charge api for bank transfers, Since we never passed the metadata does it mean we are not passing our reference to stripe?